nixos/modules/services/web-apps/movim.nix

```diff
@@ -678,7 +763,13 @@ in
 
       phpfpm.pools.${pool} =
         let
-          socketOwner = if (cfg.nginx != null) then config.services.nginx.user else cfg.user;
+          socketOwner =
+            if (cfg.h2o != null) then
+              config.services.h2o.user
+            else if (cfg.nginx != null) then
+              config.services.nginx.user
+            else
+              cfg.user;
```

Comment:
This code makes me think we have a missing assertion that the user has not set both `cfg.h2o` and `cfg.nginx`.